src/room.js

```diff
@@ -0,0 +1,37 @@
+class Room {
+
+  constructor (roomname) {
+    this.roomname = roomname;
+    this.endpoints = {};
+  }
+  addEndpoint (endpointId) {
+    this.endpoints[endpointId] = {
+      name: '',
+      permissions: 'CHAT',
+      commsid: ''
+    };
+  }
+  updateEndpointName (endpointId, Name) {
+    this.endpoints[endpointId].name = Name;
+  }
+  updateEndpointPermissions (endpointId, permissions) {
```

Comment:
This naming is slightly redundant. Can we just change these to:
+ updateUsername
+ updatePermissions
+ updateCommsId

?